### PR TITLE
221214

### PR DESCRIPTION
**Pull request by dndrks**
adds docs for new `221214` features:
- https://github.com/monome/norns/pull/1611
- https://github.com/monome/norns/pull/1616
- https://github.com/monome/norns/pull/1613
- https://github.com/monome/norns/pull/1605

leaving https://github.com/monome/norns/pull/1607 out for the moment, as TAPE seems to crash when the feature is toggled on/off quickly.

### CODE REVIEW

File: norns/reference.md [hunk 1]
```diff
@@ -110,6 +110,65 @@ function cleanup()
 end
 ```
 
+## setting minimum required version
+
+Each norns update brings new features, both for scripting and general usability. If you share a script which relies on the features of a particular norns update, you can specify a minimum version requirement which will display a message on the norns screen if the script is loaded on a previous version of the software.
+
+To specify a minimum version required, set `norns.version.required` to the **YYMMDD** value before the `init()` of your script, eg:
+
+```lua
```
Comment (tehn) [inline norns/reference.md:119]: this is a cool example but way too long to simply demonstrate the min version feature--- just an init function would do

File: norns/control-clock.md [hunk 2]
```diff
@@ -119,6 +119,27 @@ An example entry:
 
 You'll notice that these entries are identical to the ones located on the mapping screen.
 
+## keyboard
```
Comment (tehn) [inline norns/control-clock.md:122]: are there any other keyboard functions that exist that we missed? (already existed?)

might be worth mentioning that kb works for password entry/etc, or typing file names

**Review by tehn — CHANGES_REQUESTED** [review]
awesome! minor edit suggestions

File: norns/control-clock.md (hunk 2 above)
Comment (dndrks) [reply norns/control-clock.md:122]: raaaad, yes yes!

File: norns/reference.md (hunk 1 above)
Comment (dndrks) [reply norns/reference.md:119]: lol totally right, got bloaaaaty

**Review by tehn — APPROVED** [review]

**Pull request merged**